Cookbook/Proposals/MocksAndStubsNextToImplementation.md

```diff
@@ -0,0 +1,53 @@
+# Put Mocks, Stubs Next to Implementation
+
+* Author(s): Sergey Shulga
+* Review Manager: TBD
+
+## Introduction
+
+Put implementationf of mocks next to the implementation of the protocol
+
+## Motivation
+
+
+- Improves Desccoverability of mocks
```

Comment:
Usually mocks and stubs are shared between multiple tests. Discoverability will be very low in that case

Reply:
> Usually mocks and stubs are shared between multiple tests

Not sure what do you mean here?

My point here is that in order to fine a mock we would just need to go to the protocol definition

Reply:
@sergdort just imagine `MockPatientResponse`